test/utilities/notifications.ts

```diff
@@ -66,19 +56,21 @@ export async function acceptNotification(
   notificationMessage: string,
   actionName: string,
   timeout: Duration
-): Promise<void> {
+): Promise<boolean> {
   console.log(`${notificationMessage}, ${actionName}, ${timeout}`);
+  await executeQuickPick('Notifications: Show Notifications', Duration.seconds(1));
 
-  // const notification = await findNotification(notificationMessage, true, timeout);
-  // if (!notification) {
-  //   throw new Error(
-  //     `Could not take action ${actionName} for notification with message ${notificationMessage}`
-  //   );
-  // }
-
-  // const elemment = await notification.getL
-  // const actionButton = await elemment.$(`.//a[@role="button"][text()="${actionName}"]`);
-  // await actionButton.click();
+  const actionButtons = await getBrowser().findElements(
+    By.css(`div.notification-list-item-buttons-container > a.monaco-button.monaco-text-button`)
+  );
+  for (const button of actionButtons) {
+    if ((await button.getText()).includes(actionName)) {
+      log(`button ${actionName} found`);
+      await button.click();
+      return true;
+    }
+  }
+  return false;
```

Comment:
Yay, the `acceptNotification()` helper function now has the correct functionality! 💃